.github/workflows/render-stan-docs.yaml

```diff
@@ -0,0 +1,49 @@
+# Based on https://github.com/mattnotmitt/doxygen-action
+name: Render Stan documentation
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - 'inst/stan/**'
+  push:
+    branches:
+      - main
+    paths:
+      - 'inst/stan/**'
+
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+
+    - name: Doxygen Action
+      uses: mattnotmitt/doxygen-action@v1.9.2
+      with:
+        # Path to Doxyfile
```

Comment:
In the output for this I see it thinks it can't access `functions` and that no html is found at `inst/stan/html`. Thoughts @jamesmbaazam?

Reply:
Going try to debug locally with [act](https://github.com/nektos/act). There may be a weird interaction between the paths specified in the `Doxyfile` and the paths specified here. I'll check now.

Reply:
> This has been a weirdly fun exercise. I really appreciate {Roxygen2} now 😆.

Haha I know right. Try Julia Docstrings (or Python) so painful!

> I think the R stan community n

Agree they just need a better focus on user packaging across the board IMO. 

> I could explore it later. Or have you?

I haven't looked at this and that would be great if we could get it to work

Reply:
> In the output for this I see it thinks it can't access `functions` and that no html is found at `inst/stan/html`. Thoughts @jamesmbaazam?

I think this is fixed now as of 29441a2e5e83b2e02ed443d83bcde8b8752b4492.